### DISCUSSION TITLE
How to deploy in intranet? direct ip access

### DISCUSSION THREAD

**Discussion by endual** [rustdesk/rustdesk#532] [General]

**Comment by rustdesk** [comment]
Enable direct IP access from the menu on remote side RustDesk (not enabled by default). Connect to remote by input the ip (where you input ID) , no encryption in this way.

**Reply by panoskj** [reply]
Is there a way to enable encryption in this mode?

**Reply by IkelAtomig** [reply]
Yeah, It would be better option but what are the disadvantages of being unencrypted ?

**Reply by AlexPdx007** [reply]
I see the advantages wen u're connecting to intranet computers and i don't want to remember all the ID's ...all u need is the IP address of that host...and the RustDesk running just as a service.
I'm starting to use RustDesk as an alternative to Dameware: Remote Access Software ,and it would be helpful if i could create an installer.ps1 with my desires server IP, desired default password, and Direct IP connection enabled :) 
So far i could only set my Server IP and default host Password...but if there is an "enable-audio = 'Y'" setting ...i guess that there must be a "enable-direct ip = Y" option...

**Reply by Darin755** [reply]
> Yeah, It would be better option but what are the disadvantages of being unencrypted ?

Your going to be vulnerable to many different sniffing attacks. You can mitigate many of the attack vectors though a physical connection but it isnt going to be perfect

**Reply by IkelAtomig** [reply]
I think it's safe unless there are no other devices in that intranet apart from my own ?

**Reply by ameaninglessname** [reply]
I've enabled direct IP access on remote side, but it's still connected through relay server. I don't know why.
![image](https://github.com/rustdesk/rustdesk/assets/20263103/af1a5f81-e1c7-4107-bc1a-2e048398d7ad)

**Comment by wztdream** [comment]
The RustDesk connection between two computers in my home LAN is displayed in encrypted direct mode, but there is still frequent lag. I expected a direct connection to be fast. So, what could be the reasons for the lag? Do I need to do anything with the router settings or check anything specific?

**Reply by Darin755** [reply]
Are you running over WiFi? If so, try a hard wired connection.

You also could trying a ping to see latency between devices.

**Reply by wztdream** [reply]
Thank you for the suggestion.

Yes, I am running over WIFI. I will try the hard wired connection tomorrow. 

The ping results are as below, it seems to me the Maximum about 200ms is too slow right? Maybe this is caused by WiFI, will check it tomorrow.
 Minimum = 2ms, Maximum = 238ms, Average = 73ms

**Reply by Darin755** [reply]
238ms is pretty high. What's your jitter for your regular WiFi connection? It sounds like this is a WiFi related issue. 

I'm not sure how old your network hardware is but if you are looking for something to upgrade you could pickup a router with WiFi 6.

**Reply by wztdream** [reply]
My router is pretty new and has signals for both WiFi 6 and WiFi 5. Your guess is correct; my remote computer sometimes connects to the WiFi 5 signal, which may explain the delay. Now I've switched to a cable connection, and the ping is about 1ms, which is very fast. Up until now, the delay of RustDesk has also been below 5ms, most of the time around 1ms, which is very fast too.
